releaf-core/spec/lib/error_formatter_spec.rb

```diff
@@ -198,6 +224,16 @@ def base_validation
       allow(message).to receive(:data).and_return(nil)
 
       expect( I18n ).to receive(:t).with(message, scope: "activerecord.errors.messages.book").and_call_original
+
+      template = "%{class} with id %{id} has error \"error message\" on attribute \"%{attribute}\""
+      expect( I18n ).to receive(:t).with(template, {
+        default: template,
+        attribute: 'title',
+        class: 'Book',
+        id: 'null',
+        scope: "activerecord.errors.messages.book"
+      }).and_call_original
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.